### PR TITLE
First import after refactor

### PR DESCRIPTION
**Pull request by samlown**
* Related to https://github.com/invopop/ticketbai/pull/6

### CODE REVIEW

**Comment by cavalle** [comment]
@samlown I just pushed my latest work on this refactor along with the related changes in https://github.com/invopop/gobl.ticketbai/pull/2.

I delimited the scope to completing the extraction of the `gobl.ticketbai` package, update to use the latest GOBL and make sure everything works. I tested locally and managed to issue invoices successfully in the Batuz test environment.

I suggest that you review these two PRs, provide any feedback you may have, we decide whether we should address it in these PRs or in subsequent ones, we merge and I continue with the other things we've identified (e.g. move from item keys extensions…)

In summary, with this PR doesn't aim to cover everything, just make progress in the right direction (hopefully!)

**Comment by samlown** [comment]
Looking great @cavalle! And yes, getting the extensions to work with the exception cases would be a great next step. The README.md in that regard is quite out of date.

**Comment by cavalle** [comment]
Thanks @samlown! I've updated both READMEs, added a basic CLI and setup GH workflows. I'm merging this now.
